.env.test.example

```
@@ -50,46 +54,48 @@ QSTASH_NEXT_SIGNING_KEY=test-qstash-next
 # ---- AI providers / gateway ------------------------------------------------
-AI_GATEWAY_API_KEY=test-gateway-key
+AI_GATEWAY_API_KEY=
 AI_GATEWAY_URL=https://ai-gateway.vercel.sh/v1
-OPENAI_API_KEY=test-openai
-OPENROUTER_API_KEY=test-openrouter
-ANTHROPIC_API_KEY=test-anthropic
-XAI_API_KEY=test-xai
-EMBEDDINGS_API_KEY=test-embeddings
+OPENAI_API_KEY=
+OPENROUTER_API_KEY=
+ANTHROPIC_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ANTHROPIC_API_KEY key should go before the OPENAI_API_KEY key

(UnorderedKey)